src/graphql/main/mutations.ts

```diff
@@ -33,55 +33,63 @@ import CaptchaRequestAuthCodeMutation from "@graphql/root/mutation/captcha-reque
 import CaptchaCreateChallengeMutation from "@graphql/root/mutation/captcha-create-challenge"
 
 // TODO: // const fields: { [key: string]: GraphQLFieldConfig<any, GraphQLContext> }
-const fields = {
-  // unauthed
-  userRequestAuthCode: UserRequestAuthCodeMutation,
-  userLogin: UserLoginMutation,
+export const mutationFields = {
+  unauthed: {
+    userRequestAuthCode: UserRequestAuthCodeMutation,
+    userLogin: UserLoginMutation,
 
-  captchaCreateChallenge: CaptchaCreateChallengeMutation,
-  captchaRequestAuthCode: CaptchaRequestAuthCodeMutation,
+    captchaCreateChallenge: CaptchaCreateChallengeMutation,
+    captchaRequestAuthCode: CaptchaRequestAuthCodeMutation,
 
-  lnInvoiceCreateOnBehalfOfRecipient: LnInvoiceCreateOnBehalfOfRecipientMutation,
-  lnUsdInvoiceCreateOnBehalfOfRecipient: LnUsdInvoiceCreateOnBehalfOfRecipientMutation,
-  lnNoAmountInvoiceCreateOnBehalfOfRecipient:
-    LnNoAmountInvoiceCreateOnBehalfOfRecipientMutation,
+    lnInvoiceCreateOnBehalfOfRecipient: LnInvoiceCreateOnBehalfOfRecipientMutation,
+    lnUsdInvoiceCreateOnBehalfOfRecipient: LnUsdInvoiceCreateOnBehalfOfRecipientMutation,
+    lnNoAmountInvoiceCreateOnBehalfOfRecipient:
+      LnNoAmountInvoiceCreateOnBehalfOfRecipientMutation,
+  },
 
-  // authed
-  userQuizQuestionUpdateCompleted: UserQuizQuestionUpdateCompletedMutation,
-  deviceNotificationTokenCreate: DeviceNotificationTokenCreateMutation,
+  authed: {
+    noWalletId: {
```

Comment:
could also be `account` and `wallet` as a way to seggregate those two set of mutations

Reply:
I was thinking this as well when I was moving these around. Done https://github.com/GaloyMoney/galoy/pull/2097/commits/5bc7ab6fd86590e6f410a73f4cc3988d26d61c5e

Reply:
Done https://github.com/GaloyMoney/galoy/pull/2097/commits/b973aea69d3903206f09a5942c6a5c533ed8130d